plans/plan.md

```diff
@@ -0,0 +1,26 @@
+# Plans for TRBoom's Personal Website
+
+## Tutorial for a Fake News Detector Powered by Transformers
+
+
+### Overiew
+I'm building a fake news detector for another class using a transformer based approach. The website would chronical my approaches or be a tutorial on how to get a transformer up and running.
+
+
+### Goals
+Educate visiters and provide code for their own projects in the field of Machine Learning and Natural Langauge Processing.
```

Comment:
While this is a good overall general goal and mission statement for the site, you should have several specific goals each with measurable objectives for the site.